examples/Examples/Api.hs

```diff
@@ -1,27 +1,30 @@
 {-# LANGUAGE OverloadedRecordDot #-}
+{-# LANGUAGE QualifiedDo #-}
```

Comment:
We should just add this to the cabal file

Reply:
I'm still being careful about compat, but we can add just for the test-suite ig